haiku-apps/internalmidi/internalmidi-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="MIDI Node Creator"
+DESCRIPTION="Creates a MIDI (MidiKit2) node for the internal \
+General MIDI synthesizer of BeOS/Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/InternalMIDI"
+COPYRIGHT="2001-2008 Werner Freytag
+	2009 Haiku"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="a7d0ef0d1cb76ce1ba343f1d6e330007dd57c953"
+SOURCE_URI="https://github.com/HaikuArchives/InternalMIDI/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="19f8a6872ddf0fb4273f748e84cbdd7d06a76a1c23e4ef59482d002b31b36c0f"
+SOURCE_DIR="InternalMIDI-$srcGitRev"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```

Comment:
We prefer to have x86_gcc2 before x86 because most of our recipes use this order.
```diff
-ARCHITECTURES="x86 x86_gcc2 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```